[support] Replace 'could' and 'might' (#4354)
cplusplus/draft: source/support.tex

after inclusion of any member of the set of library headers
indicated in the corresponding comment in this synopsis.
\begin{note}

> Future revisions of \Cpp{} can replace

Reviewer: I know it's necessary, but this does not seem like an improvement.

Reviewer 2: How about rephrasing completely: "The values of these macros will increase in future revisions of \Cpp{} which modify the corresponding feature."?

Author: Future revisions of \Cpp{} will replace
the values of these macros with greater values
when the corresponding feature is modified.

data race\iref{res.on.data.races}.
\end{note}
\begin{note}
The order of registration can be indeterminate
if \tcode{at_quick_exit} is called from more than one thread.
\end{note}
\begin{note}
The
\tcode{at_quick_exit} registrations are distinct from the \tcode{atexit} registrations,

> and applications can call both registration functions with the same argument.

Reviewer: This seems to change the meaning. I know it's effectively the same, but it imparts different information to the reader.

Author: Tried a different approach.

the program shall also define
the corresponding version without the \tcode{size} parameter.
\begin{note}

> The default behavior below can change in the future, which will require
> replacing both deallocation functions when replacing the allocation function.

Reviewer 3: I think this has lost the intent, which is that we've identified this as a desirable change. How about:
```suggestion
It is anticipated that the default behavior will change in the future and will instead require
replacing both deallocation functions when replacing the allocation function.
```

the program shall also define
the corresponding version without the \tcode{size} parameter.
\begin{note}

> The default behavior below can change in the future, which will require

Reviewer 3: ```suggestion
It is anticipated that the default behavior will change in the future and will instead require
```
